tools/wptserve/wptserve/response.py

```diff
@@ -181,7 +181,7 @@ def iter_content(self, read_file=False):
         True, the entire content of the file will be returned as a string facilitating
         non-streaming operations like template substitution.
         """
-        if isinstance(self.content, string_types):
+        if isinstance(self.content, binary_type):
```

Comment:
I think we can just use `bytes` which is in both Py2 and 3, but not sure whether reviewers agree.